djredcap/management/subcommands/convert.py

```diff
@@ -39,10 +39,13 @@ class Command(BaseCommand):
     args = 'filename'
 
     json_filename = ''
-
+    # Add a command line option -m allowing the user to set a filename for models.
+    # Filename is relative to the current working directory.
     option_list = BaseCommand.option_list + (
         make_option('--JSON', action='store', dest='json',
-                    help='Print a JSON file when using convert, default no'),
+            help='Print a JSON file when using convert, default no'), \

```

Comment:
No need for escaping backslashes. Also indent the help to line up with the first argument to `make_option`.